backend/dataall/aws/handlers/parameter_store.py

```diff
@@ -23,11 +23,25 @@ def client(AwsAccountId, region):
     @staticmethod
     def get_parameter_value(AwsAccountId, region, parameter_path):
         if not parameter_path:
-            raise Exception('Secret name is None')
+            raise Exception('Parameter name is None')
         try:
             parameter_value = ParameterStoreManager.client(
                 AwsAccountId, region
             ).get_parameter(Name=parameter_path)['Parameter']['Value']
         except ClientError as e:
             raise Exception(e)
         return parameter_value
+
+    @staticmethod
+    def update_parameter(AwsAccountId, region, parameter_name, parameter_value):
+        if not parameter_name:
+            raise Exception('Parameter name is None')
+        if not parameter_value:
+            raise Exception('Parameter value is None')
+        try:
+            response = ParameterStoreManager.client(
+                AwsAccountId, region
+            ).put_parameter(Name=parameter_name, Value=parameter_value, Overwrite=True)['Version']
+        except ClientError as e:
+            raise Exception(e)
```

Comment:
Perhaps we can use try...except....else... to include the 'return' clause in the else block. Just a suggestion.